cmd/agent/app/flare.go

```diff
@@ -12,6 +12,8 @@ import (
 	"os"
 
 	"github.com/fatih/color"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
```

Comment:
This package seems nice, but do we want to use it? It looks like it was archived in 2020, and isn't maintained anymore

Reply:
Yeah, it's a shame that it was archived, as it seems really great. Hopefully we don't have to wait too long for the go2 version.